main.rb

```diff
@@ -0,0 +1,16 @@
+def decode(text)
+  morse_text = {
+    '.-' => 'A', '-...' => 'B', '-.-.' => 'C', '-..' => 'D', '.' => 'E', '..-.' => 'F', '--.' => 'G', '....' => 'H',
+    '..' => 'I', '.---' => 'J', '-.-' => 'K', '.-..' => 'L', '--' => 'M', '-.' => 'N', '---' => 'O', '.--.' => 'P',
+    '--.-' => 'Q', '.-.' => 'R', '...' => 'S', '-' => 'T', '..-' => 'U', '...-' => 'V', '.--' => 'W',
+    '-..-' => 'X', '-.--' => 'Y', '--..' => 'Z', '' => ' '
+  }
+  morse_array = text.split(/(\s)/)
+  morse_decode = []
+  morse_array.each do |code|
+    morse_decode.push(morse_text[code])
+  end
+  morse_decode.join
+end
+
+puts decode('.-   -... --- -..-   ..-. ..- .-.. .-..   --- ..-.   .-. ..- -... .. . ...')
```

Comment:
- You have done ell so far, but according to the project requirements, you should have 3 different methods,
-- `decode_char` that translates Morse code symbols into letters
-- `decode_word` that reuses the `decode_char` method to translate a single char symbol into a letter, combines translated chars into a single word to be returned as the final output
-- `decode` that reuses the `#decode_word` method to translate a single encoded-word into a word (DRY rule), it combines translated words into a whole sentence to be returned as the final output, please fix this in your project 🙏